streams/javahippie.json

```diff
@@ -0,0 +1,26 @@
+{
+  "twitch_handle": "javahippie",
+  "youtube_handle": "javahippie",
+  "twitter_handle": "javahippie",
+  "streamer_timezone": "CEST",
+  "schedule": [
+    {
+      "title": "BreakPoint - Functional Friday Lunch Break",
+      "start_date_utc": "2020-08-28T10:30",
+      "duration_in_minutes": 30,
+      "platform": "twitch",
+      "stream_link": "http://twitch.tv/javahippie",
```

Comment:
I consider this optional because I already set my `twitch_handle` above. Makes sense to define explicitly in case I go live on some other stream, though.

Reply:
Added the link mainly because of YouTube, the Channel Link is not tied to the user name, there. But the logic for Twitch could differ, of course. Then the `stream_link` would be optional

Reply:
Oh right, forgot about YouTube. :blush: Yeah, would be nice to remove the need for this for Twitch streamers to reduce the amount of redundant information.

Reply:
Removed from the required fields